contracts/governance/StakingRewards/StakingRewards.sol

```diff
@@ -146,34 +143,29 @@ contract StakingRewards is StakingRewardsStorage, Initializable {
 	 * @return The timestamp of last withdrawal
 	 * @return The accumulated reward
 	 */
-	function getStakerCurrentReward(bool considerMaxDuration) public view returns (uint256 withdrawalTime, uint256 amount) {
-		uint256 count;
+	function getStakerCurrentReward(bool considerMaxDuration) public view returns (uint256 lastWithdrawalInterval, uint256 amount) {
 		uint256 weightedStake;
-		uint256 lastFinalisedBlock = block.number - 1;
+		uint256 lastFinalisedBlock = block.number.sub(1);
```

Comment:
this increases gas spent
how block.timestamp can overflow?

Reply:
Mixed up with Mock Contract - fixed